components/SaveToDevice.js

```diff
@@ -153,6 +156,10 @@ componentDidMount() {
             Your report did not save successfully. Please try again later.
           </Text>
         </MaterialDialog>
+            <SafeAreaView alignItems = "center">
+            <TouchableOpacity onPress={() => Linking.openURL('https://forms.gle/aXVjxVrQU6jm3KUx6')}><Text style={{ color: 'blue' }}>Submit Feedback</Text></TouchableOpacity>
```

Comment:
Have you tried following the same technique used on the Welcome screen to force the footer to the bottom?